contracts/test/ProposalTester.sol

```diff
@@ -0,0 +1,40 @@
+//SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.11;
+
+contract ProposalTester {
```

Comment:
Can we make this contract Ownable with Multisig as owner and add modifier to testCall "onlyOnwer"?

Reply:
Not sure it's necessary, given we log the `msg.sender` and check it in tests.